ingest/gene_data_model.py

```diff
@@ -21,28 +21,28 @@
 
 
 class Gene:
-    SUBCOLLECTION_NAME = 'gene_expression'
-    COLLECTION_NAME = 'gene'
+    SUBCOLLECTION_NAME = "gene_expression"
+    COLLECTION_NAME = "gene"
```

Comment:
This should be "genes", not "gene" (for consistency with other models).

Reply:
Fixed in fd699d4